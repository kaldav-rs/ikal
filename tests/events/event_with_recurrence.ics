BEGIN:VEVENT
DTSTAMP:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
DTSTART:19960401T010000
DTEND:19960401T020000
RRULE:FREQ=DAILY;COUNT=100
EXDATE:19960402T010000Z,19960403T010000Z,19960404T010000Z
SUMMARY:A recurring event with exdates
END:VEVENT
