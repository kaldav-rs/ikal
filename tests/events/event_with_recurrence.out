Ok(
    VEvent {
        dtstamp: Naive(
            2017-02-09T19:23:58,
        ),
        uid: Text {
            params: {},
            text: "5UILHLI7RI6K2IDRAQX7O",
        },
        dtstart: DateTime(
            Naive(
                1996-04-01T01:00:00,
            ),
        ),
        class: None,
        created: None,
        description: None,
        geo: None,
        last_modified: None,
        location: None,
        organizer: None,
        priority: None,
        sequence: None,
        status: None,
        summary: Some(
            Text {
                params: {},
                text: "A recurring event with exdates",
            },
        ),
        transp: None,
        url: None,
        recurid: None,
        rrule: Some(
            Recur {
                freq: Daily,
                until: None,
                count: Some(
                    100,
                ),
                interval: None,
                by_second: [],
                by_minute: [],
                by_hour: [],
                by_day: [],
                by_monthday: [],
                by_yearday: [],
                by_weekno: [],
                by_month: [],
                by_setpos: [],
                wkst: None,
            },
        ),
        dtend: Some(
            DateTime(
                Naive(
                    1996-04-01T02:00:00,
                ),
            ),
        ),
        duration: None,
        attach: [],
        attendee: [],
        categories: [],
        comment: [],
        contact: [],
        exdate: [
            DateTime(
                Local(
                    1996-04-02T03:00:00+02:00,
                ),
            ),
            DateTime(
                Local(
                    1996-04-03T03:00:00+02:00,
                ),
            ),
            DateTime(
                Local(
                    1996-04-04T03:00:00+02:00,
                ),
            ),
        ],
        rstatus: [],
        related_to: [],
        resources: [],
        rdate: [],
        alarms: [],
        x_prop: {},
        iana_prop: {},
    },
)
